cmd/node/node.go

```diff
@@ -313,7 +318,18 @@ func (app *SpacemeshApp) initServices(nodeID types.NodeId, swarm service.Service
 	blockProducer := miner.NewBlockBuilder(nodeID, swarm, clock.Subscribe(), coinToss, msh, ha, blockOracle, atxdb.ProcessAtx, lg.WithName("blockProducer"))
 	blockListener := sync.NewBlockListener(swarm, blockValidator, msh, 2*time.Second, 4, lg.WithName("blockListener"))
 
-	nipstBuilder := nipst.NewNipstBuilder([]byte(nodeID.Key), commitmentConfig.SpaceUnit, commitmentConfig.Difficulty, 100, postClient, poetClient, lg.WithName("nipstBuilder")) // TODO: use both keys in the nodeID
+	poetListener := activation.NewPoetListener(swarm, poetDb, lg.WithName("poetListener"))
+
+	nipstBuilder := nipst.NewNipstBuilder(
+		[]byte(nodeID.Key), // TODO: use both keys in the nodeID
+		commitmentConfig.SpaceUnit,
+		commitmentConfig.Difficulty,
+		100,
```

Comment:
if your'e already refactoring this part, please add a note or use variable for the `100`

Reply:
This variable is never used and I'm not sure what it should be used for so I think I'll just remove it. We can always re-add it if/when we decide we need it.